package.json

```diff
@@ -17,7 +17,6 @@
     "@types/react-dom": "^18.0.5",
     "@types/react-redux": "^7.1.24",
     "axios": "^0.27.2",
-    "mui-image": "^1.0.7",
```

Comment:
Bye~~~
also don't forget to remove ``@types/mui-image``

Reply:
Ooh thanks! Will do!